data/pycon-au-2011/videos/sysadmins-vs-developers-a-take-from-the-other-si.json

```diff
@@ -1,26 +1,26 @@
 {
-  "id": 988, 
-  "category": "PyCon AU 2011", 
-  "slug": "sysadmins-vs-developers-a-take-from-the-other-si", 
-  "title": "Sysadmins vs Developers, a take from the other side of the fence", 
-  "summary": "A 30 minute foray into the world of the system administrator, focusing on\nsituations shared between sysadmins and developers, however differ from\nperspective, motivations and problems. The primary goal here is to provide\nsome light hearted anecdotes, examples and situations, that developers will\nrelate to from their&nbsp_place_holder;experiences&nbsp_place_holder;and\ngarner a new appreciation for the role of the sysadmin.\n\n", 
-  "description": "", 
-  "quality_notes": "", 
-  "language": "English", 
-  "copyright_text": "Creative Commons Attribution license", 
-  "thumbnail_url": "http://i.ytimg.com/vi/tyDiw7Nqu7w/hqdefault.jpg", 
-  "duration": null, 
+  "id": 988,
+  "category": "PyCon AU 2011",
+  "slug": "sysadmins-vs-developers-a-take-from-the-other-si",
+  "title": "Sysadmins vs Developers, a take from the other side of the fence",
+  "summary": "A 30 minute foray into the world of the system administrator, focusing\non situations shared between sysadmins and developers, however differ\nfrom perspective, motivations and problems. The primary goal here is to\nprovide some light hearted anecdotes, examples and situations, that\ndevelopers will relate to from\ntheir&nbsp\\_place\\_holder;experiences&nbsp\\_place\\_holder;and garner a\nnew appreciation for the role of the sysadmin.\n",
```

Comment:
`their&nbsp\_place\_holder;experiences&nbsp\_place\_holder;and g` to clean up (ideally -- not mandatory for this PR)

[Source page](http://pycon-au.org/2011/conference/schedule/event/28/index.html):

> that developers will relate to from theirÂ experiencesÂ and garner a new